src/BSeries.jl

```diff
@@ -1554,61 +1555,61 @@ function low_order_energy_preserving(trees, coefficients)
     return energy_preserving_trees_test(same_order_trees,same_order_coeffs)
 end
 
-#        remove_trunk(a)
+#        branches(a)
 
 #This function returns the forest of level_sequences 
 #obtained after removing the rightmost trunk of a given
 #level_sequence.
-function remove_trunk(a)
-    m = num_branchs(a)
-    branchs_array = Array{Array}(undef, m)
+function branches(a)
```

Comment:
It's a bit confusing to me that you have two functions named `branches` and `get_branches`.  Especially since `branches` does seem to get the branches while `get branches` actually moves the branches around.  Please add a comment explaining what `get_branches` does and then consider renaming or merging these functions so that the names are clear.

Reply:
I guess if you change `branches` back to `remove_trunk` then it resolves this issue.

Reply:
I merged the two functions into one `get_branches` function. I hope this is easier to read